cmd/kogito/command/install/kogito_operator_test.go

```diff
@@ -39,21 +39,6 @@ func Test_InstallOperator(t *testing.T) {
 	assert.Contains(t, lines, "Kogito Operator successfully deployed")
 }
 
-func Test_InstallOperatorNoNamespace(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
Ah right, this shouldn't be removed, added it back :)